### ISSUE TITLE
Skip link links to `#content`, but there's no element with that id

### ISSUE THREAD

**Issue by Kiwow** [mdn/fred#802]
### What information was incorrect, unhelpful, or incomplete?

On the search page (for an example, see [a search for `:visited`](https://developer.mozilla.org/en-US/search?q=%3Avisited)), pressing <kbd>Tab</kbd> reveals a skip link. That link's href is `#content`, but since there is no element with an id of "content", interacting with the link doesn't do anything (focus stays on the link).

### What did you expect to see?

Interacting with the skip link should move focus to the search results, i.e. the link's href should be set to the search results' container's id. I believe this should be the `section` (which contains information about how many documents were found as well as the search results), currently identified only by a class of `.site-search__results`.

This may be complicated by the `.site-search__results` element being inside the `mdn-site-search` custom element's shadow DOM.
